partiql-eval/src/main/java/org/partiql/eval/Row.java

```diff
@@ -10,14 +10,18 @@
  */
 public class Row {
 
+    private final Datum[] values;
+
     /**
-     * TODO internalize values.
+     * Get the values in this record.
+     * @return the values
      */
-    public final Datum[] values;
+    public final Datum[] getValues() {
```

Comment:
thanks for adding back the stuff reverted from prior package audit. think there's one last minor thing that was reverted for Match.java

[`ff15aa2` (#1711)](https://github.com/partiql/partiql-lang-kotlin/pull/1711/commits/ff15aa2a53c5e40ac4790c0209ddd8093adc8799#r1917175319)